src/webxdc.rs

```diff
@@ -251,6 +251,7 @@ impl Context {
                     .ok_or_else(|| format_err!("Status object expected."))?,
                 );
                 status_update.set_quote(self, Some(&instance)).await?;
+                status_update.param.remove(Param::GuaranteeE2ee); // may be set by set_quote(), if #2985 is done, this line can be removed
```

Comment:
this doesn't affect verified groups?

Reply:
i do not think so, there is an additional check at some later point. but it is good to try that out once this is merged.